RedBoxMachine.java

```diff
@@ -37,23 +39,36 @@ public RedBoxMachine()
     */
    public int searchForMovie(String title)
    {
+      for (DVD DVD : lib){
```

Comment:
It is not good practice to use the type name as the variable name. Better to say `for (DVD dvd : lib)" or even just `for (DVD d : lib)`